collect_app/src/main/java/org/odk/collect/android/activities/FormManagerList.java

```diff
@@ -193,6 +192,24 @@ protected void onPause() {
         super.onPause();
     }
 
+    @Override
+    protected void setupAdapter(String sortOrder) {
+        List<Integer> checkedInstances = new ArrayList();
```

Comment:
Why do you need to cast these to ints? That seems dangerous to me. How about leaving them as longs and then using `Arrays.asList`? Then in the usage, you can cast the ID from the db as a long.